tutorials/ExploringDeutschJozsaAlgorithm/DeutschJozsaAlgorithmTutorial_P2.ipynb

```diff
@@ -197,6 +197,25 @@
     "Note that this algorithm requires only <span style=\"color:green\">$1$</span> oracle call, and always produces the correct result."
    ]
   },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can follow the steps of the algorithm for the constant and the balanced scenarios using a neat visualization. Since Deutsch algorithm deals only with states with real amplitudes, we can map all states on the unit circle, and follow the state evolution through the steps.\n",
+    "\n",
+    "1-2. Start with a qubit in the $|0\\rangle$ state and apply the H gate to the qubit.\n",
+    "   <img width=60% src=\"./img/2-plus-state.png\"/>\n",
```

Comment:
Can we provide some `alt` text for improved accessibility?

Reply:
Do you think this will improve user experience? The images are effectively a representation of what was said in lines 177-186, I'm not sure I'd be able to describe them in other ways. Do you have thoughts on how to do it without just repeating the text?